models/points.go

```diff
@@ -1137,7 +1141,10 @@ func (t Tags) HashKey() []byte {
 	for k, v := range t {
 		ek := escapeTag([]byte(k))
 		ev := escapeTag([]byte(v))
-		escaped[string(ek)] = string(ev)
+
+		if len(string(ev)) > 0 {
```

Comment:
is there any reason to cast this to a string?  `len(ev)` will get you the same answer I believe.